feature/security/gnsi/acctz/tests/record_history_truncation/record_history_truncation_test.go

```diff
@@ -0,0 +1,51 @@
+package record_history_truncation_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/gnsi/acctz"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func TestAccountzRecordHistoryTruncation(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+
+	systemState := gnmi.Get(t, dut, gnmi.OC().System().State())
+
+	bootTime := systemState.GetBootTime()
+
+	// try to get records from 1 day prior to device's boot time
```

Comment:
Same comment about comments as in Accountz=3.1

Reply:
Fixed